static/src/javascripts/projects/commercial/modules/dfp/display-lazy-ads.js

```diff
@@ -19,8 +21,18 @@ const instantLoad = (): void => {
 };
 
 const displayLazyAds = (): void => {
+    const isInLazyLoadTest = isInVariantSynchronous(
+        commercialGptLazyLoad,
+        'variant'
+    );
+
+    if (isInLazyLoadTest) {
+        window.googletag.pubads().enableLazyLoad();
```

Comment:
`enableLazyLoad` takes an optional config object, by passing no config object we'll use Google's defaults which are "tuned over time", so should hopefully be optimal 🤞

Further details on the config object are available [here](https://developers.google.com/doubleclick-gpt/reference#googletag.PubAdsService_enableLazyLoad).